docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -256,3 +256,30 @@ const theme = createTheme({
   },
 });
 ```
+
+## Deprecated parameters used on translation keys functions
+
+If you are using a Picker with controlled value and views and you need to compose some of the following translation keys: `openDatePickerDialogue`, `openTimePickerDialogue`, or `clockLabelText`, after upgrading to v8 you only need to pass the formatted value and the time view (only for `clockLabelText`):
```

Comment:
> After https://github.com/mui/mui-x/pull/14505 it was possible to call those methods by passing utils as an empty object, wasn't it ?

OK you are right, for the few person (probably only the author of the issue for which we added `formattedValue`) that were using `translations.openTimePickerDialogue` and equivalents, there is a  BC and it can be worth adding the `diff`.

But the main BC remain the other way around :+1: